src/apps/stable/routes/user/userprofile.tsx

```diff
@@ -102,7 +102,7 @@ const UserProfile: FunctionComponent = () => {
             const target = evt.target as HTMLInputElement;
             const file = (target.files as FileList)[0];
 
-            if (!file || !file.type.match('image.*')) {
+            if (!/image.*/.exec(file?.type)) {
```

Comment:
I find the dedicated check for `!file` a lot easier to read personally :man_shrugging: 

```suggestion
            if (!file || !/image.*/.exec(file.type)) {
```

Reply:
Yeah, I think it's better.
@thornbill